src/com/owncloud/android/ui/activity/Preferences.java

```diff
@@ -426,7 +424,7 @@ public boolean onPreferenceClick(Preference preference) {
                     @Override
                     public boolean onPreferenceClick(Preference preference) {
                         String feedbackMail = (String) getText(R.string.mail_feedback);
-                        String feedback = (String) getText(R.string.prefs_feedback) +
+                        String feedback = (String) getText(R.string.drawer_feedback) +
```

Comment:
You could get rid of this preference

Reply:
As we spoke about it we came to the conclusion to keep this.